src/commands/optimize.ts

```diff
@@ -14,7 +14,7 @@ const { writeFile } = promises;
 export enum Optimizations {
   REMOVE_COMPONENTS='remove-components',
   REUSE_COMPONENTS='reuse-components',
-  MOVE_TO_COMPONETS='move-to-components'
+  MOVE_TO_COMPONENTS='move-to-components'
```

Comment:
![john-cena-deformed-gif](https://github.com/smoya/cli/assets/1083296/b72b2177-0564-4269-ba9f-3389ef975edb)

Reply:
It was there since a loooong time ago 🙈